src/bindings/python/src/openvino/frontend/tensorflow/node_decoder.py

```diff
@@ -0,0 +1,152 @@
+                    # does not copy data
+                    self.m_parsed_content = variable_value.value().__array__()
+
+    def get_op_name(self) -> str:
+        return self.m_operation.name
+
+    def get_op_type(self) -> str:
+        if self.m_operation.type == "Placeholder":
+            type_attr = tf.dtypes.DType(self.m_operation.node_def.attr["dtype"].type)
+            if type_attr.name == "resource" and not self.m_inner_graph:
+                if TFGraphNodeDecoder.get_variable(self.m_operation) is not None:
+                    return "Const"
+                raise Exception("Could not get variable for resource Placeholder {0}".format(self.m_operation.name))
+        return self.m_operation.type
+
+    @staticmethod
+    def get_variable(operation):
+        tf_graph = operation.graph
+        if not hasattr(tf_graph, "captures"):
+            return None
+        for var_tensor, op_tensor in tf_graph.captures:
+            if operation.outputs[0].name == op_tensor.name:
+                resource_name = var_tensor._name
+                for variable_value in operation.graph.variables:
+                    if variable_value.name == resource_name:
+                        return variable_value
+                return None
+        return None
+
+    def get_attribute(self, name):
+        if name == "shape" or name == "_output_shapes":
+            if self.m_operation.node_def.attr["shape"].shape.unknown_rank:
+                return OVAny(PartialShape.dynamic())
+            shape_dims = self.m_operation.node_def.attr["shape"].shape.dim
+            shape = [dim.size for dim in shape_dims]
+            type_num = self.m_operation.node_def.attr["dtype"].type
+            if type_num is not None and tf.dtypes.DType(type_num).name == "resource":
+                if self.m_inner_graph:
+                    return OVAny(PartialShape.dynamic())
+                variable_value = TFGraphNodeDecoder.get_variable(self.m_operation)
+                return OVAny(PartialShape(list(variable_value.shape)))
+            return OVAny(PartialShape(shape))
+        if name == "dtype":
+            type_num = self.m_operation.node_def.attr["dtype"].type
+            if tf.dtypes.DType(type_num).name == "resource":
+                if not self.m_inner_graph:
+                    variable_value = TFGraphNodeDecoder.get_variable(self.m_operation)
+                    return OVAny(tf_type_to_ov_type(variable_value.dtype))
+                else:
+                    return OVAny(Type.undefined)
+            return OVAny(tf_type_to_ov_type(type_num))
+
+        if name == "value":
+            if self.m_parsed_content.size == 1:
+                if isinstance(self.m_parsed_content, np.ndarray):
+                    return OVAny(Tensor(self.m_parsed_content))
+                return OVAny(Tensor(np.array([self.m_parsed_content]), shape=[1]))
+            ov_tensor = Tensor(self.m_parsed_content, shared_memory=True)
```

Comment:
Will it work for string tensors? We wrap such values to OVAny<std::vector<std::string>>.
Can be fixed in the next PR.

Reply:
When `value` is accessed from `Const` translator, `get_attribute` is expected to return `Tensor`. 
https://github.com/openvinotoolkit/openvino/blob/90a0e5f81a4bf14c7b0b953448fb5a6752dcec3f/src/frontends/tensorflow_common/src/op/const.cpp#L30

So if I return `OVAny<std::vectorstd::string>` the translator fails as it cannot cast `string` to `Tensor`.

And `Tensor` has no constructor from `string`, so it looks like we need to implement `Tensor(<string>)`. 

It looks like a big change, so I suggest to implement it in a separate PR.